src/mam4xx/mo_drydep.hpp

```diff
@@ -25,10 +26,111 @@ constexpr Real rair = 287.04;
 constexpr Real grav = 9.81;
 constexpr Real karman = 0.4;   // from shr_const_mod.F90
 constexpr Real tmelt = 273.15; // from shr_const_mod.F90 via physconst.F90
-
+constexpr Real r2d = 180.0 / haero::Constants::pi; // degrees to radians
 // nddvels is equal to number of species in dry deposition list for gases.
 constexpr int nddvels = mam4::seq_drydep::n_drydep;
 
+/**
+ * Finds the season index for each longitude point based on the most frequent
+ * season in the 11 vegetation classes to mitigate banding issues in dvel.
+ * @brief Finds the season index based on the given latitude.
+ *
+ * @param[in] clat_j Latitude in degrees from the host model.
+ * @param[in] lat_lai Latitude values from the NC file.
+ * @param[in] nlat_lai Size of lat_lai.
+ * @param[in] wk_lai Season_wes from the NC file.
+ * @param[out] index_season_lai Outputs the season indices.
+ */
+
+// find_season_index_at_clat only needs to be executed one time and is small.
+// Thus, execute it only on the host.
+using View1DHost = DeviceType::view_1d<Real>::HostMirror;
+using ConstView1DHost = DeviceType::view_1d<const Real>::HostMirror;
+using View1DIntHost = DeviceType::view_1d<int>::HostMirror;
+using View2DIntHost = DeviceType::view_2d<int>::HostMirror;
+using View3DIntHost = DeviceType::view_3d<int>::HostMirror;
+
+using KTH = ekat::KokkosTypes<ekat::HostDevice>;
+
+inline void find_season_index_at_lat(const Real clat_j,
```

Comment:
Does this work without a KOKKOS_INLINE_FUNCTION  annotation? It's called in a `parallel_for` below.

Reply:
Yes, it does, because this find_season_index_at_lat is only invoked from the host

Reply:
Ah, I see the KTH now. Thanks!